README.md

```diff
@@ -1,6 +1,64 @@
 # TVM RFCs
```

Comment:
Please add ASF header to the markdown, we might consider introducing similar lint checks to this repo later

Reply:
Is there a link I can use as a reference?

Reply:
https://raw.githubusercontent.com/apache/tvm/main/README.md